jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBStatement.java

```diff
@@ -358,7 +349,7 @@ private boolean executeSQL(String sql) throws TException, SQLException {
 
   private int[] executeBatchSQL() throws TException, SQLException {
```

Comment:
Why the queryId is not set in this sql function

Reply:
queryId is mainly for the resource control of data queries, which are not allowed in the batch statement, so it is not necessary to set it in batch execution.